spec/datetimeformat.html

```diff
@@ -200,10 +208,13 @@ <h1>BasicFormatMatcher ( _options_, _formats_ )</h1>
           1. For each _property_ shown in <emu-xref href="#table-datetimeformat-components"></emu-xref>, do
             1. Let _optionsProp_ be _options_.[[&lt;_property_&gt;]].
             1. Let _formatProp_ be _format_.[[&lt;_property_&gt;]].
-            1. If _optionsProp_ is *undefined* and _formatProp_ is not *undefined*, then decrease _score_ by _additionPenalty_.
-            1. Else if _optionsProp_ is not *undefined* and _formatProp_ is *undefined*, then decrease _score_ by _removalPenalty_.
+            1. If _optionsProp_ is *undefined* and _formatProp_ is not *undefined* and _formatProp_ is not `0`, then decrease _score_ by _additionPenalty_.
```

Comment:
@anba Do you think I should remove "and _formatProp_ is not `0`" here?

Reply:
Yes, when we avoid `0` and instead use `undefined`, we don't need any special-casing for `0` in BasicFormatMatcher.